addon/adapters/user.js

```diff
@@ -1,4 +1,20 @@
 import OsfAdapter from './osf-adapter';
+import Ember from 'ember';
 
 export default OsfAdapter.extend({
+    findHasMany(store, snapshot, url, relationship) {
```

Comment:
We are using Ember-data 2.3.0 in Ember-OSF and ember-data 2.7.0 in Ember-preprints.  To make use of these features, would Ember-OSF need to be updated to 2.7?

The reason I am doing this -  ember doesn't let you pass query params to relationships.  so we are using the `ember-data-has-many-query` addon. To use this addon, query params are passed in a dictionary.   The only problem is, for embeds, the keys are the same, and I need to embed more than one relationship. I am open to a better way of doing this - but I think I am going to need to build the url myself.